### ISSUE TITLE
Error while attempting to connect to a target using ssh 

### ISSUE THREAD

**Issue by rafluan** [OSSystems/docker-yocto-env#3]
The following error occurs:
`ssh: connect to host 192.168.7.2 port 22: Connection refused`

**Comment by otavio** [comment]
Indeed. The container does not expose those ports so we'll need to extend the environment to support it.
